Source/WebCore/platform/graphics/avfoundation/MediaSelectionGroupAVFObjC.h

```diff
@@ -90,7 +90,7 @@ class MediaSelectionGroupAVFObjC : public RefCounted<MediaSelectionGroupAVFObjC>
     RetainPtr<AVPlayerItem> m_playerItem;
     RetainPtr<AVMediaSelectionGroup> m_mediaSelectionGroup;
     OptionContainer m_options;
-    MediaSelectionOptionAVFObjC* m_selectedOption { nullptr };
+    RefPtr<MediaSelectionOptionAVFObjC> m_selectedOption;
```

Comment:
This looks risky? Are we sure this cannot introduce a reference cycle?

Reply:
Good point. I'll change this to WeakPtr.